specs-go/config.go

```diff
@@ -25,6 +25,8 @@ type Spec struct {
 	Linux Linux `json:"linux" platform:"linux,omitempty"`
 	// Solaris is platform specific configuration for Solaris containers.
 	Solaris Solaris `json:"solaris" platform:"solaris,omitempty"`
+	// Windows is the platform specific configuration for Windows based containers.
+	Windows Windows `json:"windows" platform:"windows,omitempty"`
```

Comment:
This is matching a broken pattern ;).  See #502 for what I think we should be using instead.